targets/targets.json

```diff
@@ -3132,7 +3132,7 @@
         "device_name": "NUC472HI8AE",
         "bootloader_supported": true
     },
-    "NCS36510": {
+    "36510": {
```

Comment:
@maclobdell I have already pushed this, and on my end it shows up in the commits to this PR.